### ISSUE TITLE
P value "stars"

### ISSUE THREAD

**Issue by hadley** [hadley/emo#1]
From `printCoefMat()`:

```
 Signif <- symnum(pv, corr = FALSE, na = FALSE, 
                  cutpoints = c(0, 0.001, 0.01, 0.05, 0.1, 1), 
                  symbols = c("***", "**", "*", ".", " "))
```

**Comment by hadley** [comment]
And `?format.pval`

**Comment by hadley** [comment]
https://twitter.com/michaelhoffman/status/636172115729887232 + https://twitter.com/neuhier_/status/857173902816145411

**Comment by hadley** [comment]
Fixed in 4be1aa3
